lib/sorbet-rails/model_plugins/active_record_finder_methods.rb

```diff
@@ -47,6 +47,24 @@ def create_finder_methods_for(class_rbi, class_method:, include_methods_that_ret
         return_type: self.model_class_name,
         class_method: class_method
       )
+      class_rbi.create_method(
```

Comment:
These methods available on ActiveRecord relation as well right? Can you add sig to the bundled RBI?

https://github.com/chanzuckerberg/sorbet-rails/blob/master/lib/bundled_rbi/active_record_relation.rbi